mne/preprocessing/ica.py

```diff
@@ -1154,6 +1161,12 @@ def find_bads_ecg(self, inst, ch_name=None, threshold=None, start=None,
             If True, data annotated as bad will be omitted. Defaults to True.
 
             .. versionadded:: 0.14.0
+        measure : {'zscore', "cor"}
```

Comment:
```suggestion
        measure : 'zscore' | 'cor'
```
This is how we document specific allowed string values